src/tasks/AndroidAppBuilder/ApkBuilder.cs

```diff
@@ -342,6 +343,11 @@ public ApkBuilder(TaskLoggingHelper logger)
             }
         }
 
+        if (ProfileAOT)
+        {
+            defines.AppendLine("add_definitions(-DPROFILE_AOT=1)");
+        }
```

Comment:
I'd prefer if we called the option `ForceFullAOT` and the define `-DFULL_AOT=1` since we only use this to switch between normal AOT and full AOT mode and there's nothing that ties this to profiled AOT (i.e. you could decide to AOT some random assemblies without using a profile).